src/desugaring.jl

```diff
@@ -1864,23 +1898,6 @@ end
 #-------------------------------------------------------------------------------
 # Expand for loops
 
-# Extract the variable names assigned to from a "fancy assignment left hand
-# side" such as nested tuple destructuring.
-function foreach_lhs_var(f::Function, ex)
```

Comment:
FWIW, `foreach_lhs_var` was written intentionally in this slightly-weird way because we rarely care to allocate the list of lhs vars, we just want to iterate over them.